shell/platform/linux/fl_engine_private.h

```diff
@@ -276,6 +276,42 @@ void fl_engine_send_mouse_pointer_event(FlEngine* engine,
                                         double scroll_delta_y,
                                         int64_t buttons);
 
+void OnPointerDown(FlEngine* self,
```

Comment:
FlEngine shouldn't contain handlers like this, instead look at `fl_scrolling_manager.cc` as an example of how to handle this in a separate class.